src/js/providers/html5.js

```diff
@@ -227,18 +227,19 @@ define([
         }
 
         function _checkVisualQuality() {
-            if (_visualQuality.width !== _videotag.videoWidth || _visualQuality.height !== _videotag.videoHeight) {
-                _visualQuality.width = _videotag.videoWidth;
-                _visualQuality.height = _videotag.videoHeight;
-                if (!_visualQuality.width || !_visualQuality.height) {
```

Comment:
Also up top make sure to add levels to the initial definition of _visualQuality so we never get a 'levels undefined' error
`var _visualQuality = {
  levels: {}
},`